pages/reference/public-api/overview.mdx

```diff
@@ -6,3 +6,4 @@ Avo API returns JSON encoded responses and uses standard HTTP response codes.
 Avo API Uses [Basic Authentication](https://en.wikipedia.org/wiki/Basic_access_authentication) to authenticate requests.
 
 - [Export Tracking Plan](/public-api/export-tracking-plan): Export your Avo tracking plan as JSON from any branch
+- [Export Branch Stats](/public-api/export-branch-stats): Export your Avo branch stats as CSV for all branches in the workspace
```

Comment:
I wonder if we should just call it Branches? (if you agree, let's change the header in `/export-branch-stats.mdx` as well)
```suggestion
- [Export Branches](/public-api/export-branch-stats): Export your Avo branch stats as CSV for all branches in the workspace
```